src/Controller/Backend.php

```diff
@@ -59,11 +59,11 @@ public function before(Request $request, Application $app)
     {
         /** @var LabelsExtension $extension */
         $extension = $app['extensions']->get('Bolt/Labels');
-        $dir = $extension->getWebDirectory()->getPath();
+        $dir = $app['paths']['root'] . $extension->getWebDirectory()->getPath();
```

Comment:
> What would be the appropriate approach?

Should be just:
```
$extension->getWebDirectory()->getPath()
```

But yes, ResourceManager is needed there … note that it is on its way out too as its almost all in `bolt/filesystem` land now :smile: